src/Frontend/Core/Engine/Model.php

```diff
@@ -326,6 +329,29 @@ public static function getModuleSettings($module)
     }
 
     /**
+     * Creates router for module based on routing file
+     *
+     * @param $module
+     * @return null|Router
+     */
+    public static function getModuleRouter($module)
```

Comment:
I'd put this method in the `Frontend/Core/Engine/Base/Config` class, it's only used there, so it would be more logical to be in there. This would also enable you to make the method not static.